frontend/src/App.tsx

```diff
@@ -12,6 +14,7 @@ function App() {
         <Route path="/" element={<Home />} />
         <Route path="google" element={<GoogleAuthLogin />} />
       </Routes>
+      <Footer Logo={Logo} />
```

Comment:
No need to pass the logo as a prop, instead use it directly in the footer components as the logo is not dynamically changing.